OutOfSchool/OutOfSchool.WebApi/Services/ChildService.cs

```diff
@@ -26,10 +29,11 @@ public class ChildService : IChildService
         /// <param name="childRepository">Repository for the Child entity.</param>
         /// <param name="parentRepository">Repository for the Parent entity.</param>
         /// <param name="logger">Logger.</param>
-        public ChildService(IEntityRepository<Child> childRepository, IParentRepository parentRepository, ILogger<ChildService> logger)
+        public ChildService(IEntityRepository<Child> childRepository, IParentRepository parentRepository, IApplicationRepository applicationRepository, ILogger<ChildService> logger)
```

Comment:
Class public API was changed (new constructor parameter and new method added), thus tests should be updated as well

Reply:
It will be done in scope of another task